### PR TITLE
modify date

### PR DESCRIPTION
**Pull request by AlexFloresVeliz**
Made changes to the date @AmadoSuarez

### CODE REVIEW

**Review by inehemiasm — APPROVED** [review]
Great!

**Review by JoanQuintero — APPROVED** [review]
Makes sense

**Comment by manybreak22** [comment]
Awesome

File: requirements.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ Version 1.0
 
 Team 5
 
-10/01/2018
+10/10/2018
```
Comment (AmadoSuarez) [inline requirements.md:10]: Yeah. It had the old date

**Review by icedalbert — APPROVED** [review]

File: requirements.md (hunk 1 above)
Comment (AmadoSuarez) [reply requirements.md:10]: 🚢

**Review by lajarashany — APPROVED** [review]

**Review by salam4 — APPROVED** [review]
👍

**Review by manybreak22 — APPROVED** [review]
I approve ✨

**Pull request merged**